routes/collectionPages.js

```diff
@@ -203,10 +203,10 @@ async function renameCollectionPage (req, res, next) {
 
 router.get('/:siteName/collections/:collectionName', attachRouteHandlerWrapper(listCollectionPages))
 router.get('/:siteName/collections/:collectionName/pages', attachRouteHandlerWrapper(listCollectionPagesDetails))
-router.post('/:siteName/collections/:collectionName/pages', attachRouteHandlerWrapper(createNewcollectionPage))
+router.post('/:siteName/collections/:collectionName/pages', attachRollbackRouteHandlerWrapper(createNewcollectionPage))
 router.get('/:siteName/collections/:collectionName/pages/:pageName', attachRouteHandlerWrapper(readCollectionPage))
 router.post('/:siteName/collections/:collectionName/pages/:pageName', attachRouteHandlerWrapper(updateCollectionPage))
-router.delete('/:siteName/collections/:collectionName/pages/:pageName', attachRouteHandlerWrapper(deleteCollectionPage))
+router.delete('/:siteName/collections/:collectionName/pages/:pageName', attachRollbackRouteHandlerWrapper(deleteCollectionPage))
 router.post('/:siteName/collections/:collectionName/pages/:pageName/rename/:newPageName', attachRouteHandlerWrapper(renameCollectionPage))
```

Comment:
`renameCollectionPage` should use `attachRollbackRouteHandlerWrapper` as well in case the creation of the new file succeeds but the delete of the old file fails

Reply:
Resolved in https://github.com/isomerpages/isomercms-backend/pull/91/commits/967eaa01b0087bbd047e511dc28ccc8993a1eadf